pkg/prometheus/promcfg.go

```diff
@@ -1102,6 +1102,11 @@ func (cg *ConfigGenerator) BuildCommonPrometheusArgs() []monitoringv1.Argument {
 		{Name: "config.file", Value: path.Join(ConfOutDir, ConfigEnvsubstFilename)},
 	}
 
+	if cg.version.Major >= 3 {
+		// remove web.console.templates and web.console.libraries if prometheus version is 3.0.0 or greater
+		promArgs = promArgs[len(promArgs)-1:]
+	}
```

Comment:
(suggestion) I would rather reverse the test 
```suggestion
	if cg.version.Major == 2 {
		// Add web.console.templates and web.console.libraries only if Prometheus version is v2.x.
		promArgs = append(promArgs, {Name: "web.console.templates", Value: "/etc/prometheus/consoles"}, ...)
	}
```

Reply:
Flipped the condition